### PR TITLE
ansible-lint: 6.5.2 -> 6.7.0

### PR DESCRIPTION
**Pull request by stasjok**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tjni** [comment]
This has been updated to 6.5.2 already on `staging-next`. I'm not sure the best way to resolve this if you need 6.7.0 ASAP but thought I would let you know.

File: pkgs/development/python-modules/ansible-lint/default.nix [hunk 1]
```diff
@@ -20,13 +22,13 @@
 
 buildPythonPackage rec {
   pname = "ansible-lint";
-  version = "6.4.0";
+  version = "6.7.0";
   format = "pyproject";
   disabled = pythonOlder "3.6";
```
Comment (tjni) [inline pkgs/development/python-modules/ansible-lint/default.nix:27]: This is incidental, so from my point of view you don't have to address it in this PR, but I saw that, from 6.0.0 onwards, ansible-lint only formally supports Python versions 3.8 and greater (https://github.com/ansible/ansible-lint/pull/1850).

**Comment by mweinelt** [comment]
Probably best to wait for staging-next to reach master within the next 48 hours.

**Review by sengaya — APPROVED** [review]
looks good, tested on macOS

File: pkgs/development/python-modules/ansible-lint/default.nix [hunk 2]
```diff
@@ -42,7 +44,9 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     ansible-compat
     ansible-core
+    black
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ansible-lint/default.nix:47]: Is not black a runtime dependency, too?!

Comment (mweinelt) [reply pkgs/development/python-modules/ansible-lint/default.nix:47]: Yes, it is. That's why its propagated.

**Pull request merged**